os/fs/smartfs/Kconfig

```diff
@@ -95,6 +95,12 @@ config SMARTFS_JOURNALING
                 of file descriptors allowed and sector size defined. Thus, to
                 minimize the area reserved for journaling, it is advised to keep
                 sector size small.
+
+config SMARTFS_DOWNLOAD_BIN
```

Comment:
I don't know why this is necessary.

Reply:
If this config is not present, it always tries to search for user binary and downloads it.

Reply:
When `user` option is inputted. Not always. `user` should not be flashed even if `all` is inputted.
I think `user` should be working only with `user` option.